### PR TITLE
feat: add screencast and new commands into read.me

### PR DESCRIPTION
**Pull request by juandiegocv27**
# Enhance README with Screencast and Additional Commands

- Adding a [video tutorial](https://www.loom.com/share/2b5b4e61d8704fdd8ed402b92033784d?sid=ec126de5-b488-4d5a-9f0c-f55c431bb575) to help newcomers understand the project workflow, from setup to deployment.
- Including troubleshooting tips for function calls failing due to insufficient gas fees.
- Updating usage instructions with detailed commands for testing, building, and deploying NEAR smart contracts.

### Changes

1. **Usage Instructions:**
    - Added `cargo test` before `cargo near build`.
    - Included commands for `cargo near`, `cargo near build`, `cargo near create-dev-account`, `cargo near deploy`, and `cargo install near-cli-rs`.
    - Updated `near` command for contract selection and function calls.

2. **Video Tutorial:**
    - Added a link to the video tutorial.
    - Noted that the video shows a function call failure due to insufficient gas fees.

3. **Troubleshooting Section:**
    - Added tips for handling insufficient gas fees during function calls.
    - Provided examples for allocating sufficient gas and checking account balance.

related issue: [issue #1219](https://github.com/near/near-sdk-rs/issues/1219)

### CODE REVIEW

**Comment by juandiegocv27** [comment]
@frol ready for feedback!

File: README.md [hunk 1]
```diff
@@ -259,6 +259,9 @@ $ cargo install --path cargo-near
 ```
 </details>
 
+Got it. Here is the entire section formatted as markdown:
+
+```markdown
```
Comment (frol) [inline README.md:264]: chatgpt...

**Review by frol — CHANGES_REQUESTED** [review]
Thanks for your attempt, but the result is not engaging. We need someone who can record and edit a video with nice transitions, highlights, and at least captions + music or voice-over.

**Comment by juandiegocv27** [comment]
> Thanks for your attempt, but the result is not engaging. We need someone who can record and edit a video with nice transitions, highlights, and at least captions + music or voice-over.

Ok sure, let me fix it and create a more engaging video 🔥🫡

**Comment by Jemiiah** [comment]
@frol  i would love to work on this  if you allow it🌚

File: README.md (hunk 1 above)
Comment (juandiegocv27) [reply README.md:264]: what´s that?, im going to fix ir

**Comment by frol** [comment]
@Jemiiah Oh, I missed your comment. If you are still interested in it, please, go ahead!

**Comment by Jemiiah** [comment]
Hi Vlad,

Thanks for reaching out. I'm still interested and will proceed as
suggested. Excited to contribute further.

**Comment by juandiegocv27** [comment]
@frol Ill finish this issue before tommorrow morning (less than 16 hours)

**Comment by frol** [comment]
@juandiegocv27 Any updates on this front?

**Comment by juandiegocv27** [comment]
> @juandiegocv27 Any updates on this front?

im uploaading the video in youtube, finish the editing, in a couple of hours I'll put in the read.me

**Comment by juandiegocv27** [comment]
@frol finished and ready to merge, let me know

**Comment by juandiegocv27** [comment]
@frol sorry for the delay, editing videos is not my strong? can you assign this issue to someone else please

**Pull request closed without merging**